edit/util.js

```diff
@@ -121,3 +121,16 @@ function sectionsToMozFormat(style) {
 function clipString(str, limit = 100) {
   return str.length <= limit ? str : str.substr(0, limit) + '...';
 }
+
+// cache the first call
+function cacheFn(fn) {
```

Comment:
It may be better to rename to `cacheFirstCall`.

Reply:
yeah